models/userDao.js

```diff
@@ -21,7 +19,6 @@ const createUser = async (
   password,
   email
 ) => {
-  console.log('createUser :', name, phone_number, password);
   await prisma.$queryRaw`
   INSERT INTO users(name, gender, phone_number, nickname, password, email) 
   VALUES(${name}, ${gender}, ${phone_number}, ${nickname}, ${password}, ${email});
```

Comment:
```suggestion
  INSERT INTO
    users(
      name,
      gender,
      phone_number,
      nickname, 
      password,
      email) 
  VALUES(${name}, ${gender}, ${phone_number}, ${nickname}, ${password}, ${email});
```

좀 더 보기 좋게 바꿔주시면 좋을거 같습니다.